lib/hammer_cli_csv/content_hosts.rb

```diff
@@ -159,101 +159,57 @@ def import_locally
-                'name' => name,
-                'organization_id' => foreman_organization(:name => line[ORGANIZATION]),
-                'lifecycle_environment_id' => lifecycle_environment(line[ORGANIZATION], :name => line[ENVIRONMENT]),
-                'content_view_id' => katello_contentview(line[ORGANIZATION], :name => line[CONTENTVIEW]),
-                'facts' => facts(name, line),
-                'installed_products' => products(line),
-                'service_level' => line[SLA],
-                'type' => 'system'
-            })['id']
-            @existing[name] = host_id
+            host = @api.resource(:host_subscriptions).call(:create, params)
+            @existing[name] = host['id']
           else
-            # TODO: remove passing facet IDs to update
-            # Bug #13849 - updating a host's facet should not require the facet id to be included in facet params
-            #              http://projects.theforeman.org/issues/13849
             print(_("Updating content host '%{name}'...") % {:name => name}) if option_verbose?
-            host_id = @api.resource(:hosts).call(:update, {
-                'id' => @existing[name][:host],
-                'host' => {
-                    'name' => name,
-                    'content_facet_attributes' => {
-                        'id' => @existing[name][:content],
-                        'lifecycle_environment_id' => lifecycle_environment(line[ORGANIZATION], :name => line[ENVIRONMENT]),
-                        'content_view_id' => katello_contentview(line[ORGANIZATION], :name => line[CONTENTVIEW])
-                    },
-                    'subscription_facet_attributes' => {
-                        'id' => @existing[name][:subscription],
-                        'facts' => facts(name, line),
-                        # TODO: PUT /hosts subscription_facet_attributes missing "installed_products"
-                        # http://projects.theforeman.org/issues/13854
-                        #'installed_products' => products(line),
-                        'service_level' => line[SLA]
-                    }
-                }
-            })['host_id']
+            params['host_id'] = @existing[name]
+            host = @api.resource(:host_subscriptions).call(:update, params)
           end
 
           if line[VIRTUAL] == 'Yes' && line[HOST]
             raise "Content host '#{line[HOST]}' not found" if !@existing[line[HOST]]
             @hypervisor_guests[@existing[line[HOST]]] ||= []
-            @hypervisor_guests[@existing[line[HOST]]] << @existing[name]
+            @hypervisor_guests[@existing[line[HOST]]] << "#{line[ORGANIZATION]}/#{name}"
           end
 
-          update_host_facts(host_id, line)
-          update_host_collections(host_id, line)
-          update_subscriptions(host_id, line)
+          #update_host_facts(host, line)
+          #update_host_collections(host, line)
+          #update_subscriptions(host, line)
```

Comment:
I say just remove the code if you want it to go away, we can look at the git log to add it back later if needed.